### DISCUSSION TITLE
Hide Tabs

### DISCUSSION THREAD

**Discussion by nicfv** [squidfunk/mkdocs-material#6571] [Q&A] [answered]
Hi, I've enabled the [tabs feature](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#navigation-tabs) but would like to hide it on certain pages. Is this currently possible, and if so, how? I tried this at the top of my markdown page, but it doesn't seem to work:

```md
---
hide:
  - navigation
  - toc
  - tabs
---

# Example

...
```

**Comment by kamilkrzyskow** [comment, marked as answer]
Same answer as for your search question: https://github.com/squidfunk/mkdocs-material/discussions/6572#discussioncomment-7980380
The only difference would be different CSS or another template partial file.
```css
.md-tabs {
  display: none;
}
```
or `partials/tabs.html`.

**Reply by nicfv** [reply]
Perfect, thank you so much for your response. This is exactly what I was looking for.
